app/audit/crypto.test.ts

```diff
@@ -125,17 +127,28 @@ describe("encryptData", () => {
   it("should prepend encryption scheme version", () => {
     const plaintext = "super secret message";
 
-    const encrypted = encryptData(plaintext, PUBLIC_KEY);
+    const encrypted = encryptData(plaintext, AuditLogScheme.V2);
 
-    expect(encrypted.slice(0, 2)).toEqual("01");
+    expect(encrypted.slice(0, 2)).toEqual("02");
```

Comment:
I am wondering whether we also want to test for V1? Especially because we are probably using the same code to potentially re-encrypt the data?

Reply:
See my explanation above. That would be part of the migration logic IMHO.

Reply:
> See my explanation above. That would be part of the migration logic IMHO.

I am not sure because the functions till is quite general and would allow both or not?

Reply:
I added a new test and also an additional key pair. The plan I had in mind was to delete all traces of v1 once the migration is through, but feel free to go whichever way you find best.